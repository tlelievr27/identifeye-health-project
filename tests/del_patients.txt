ADD PATIENT 123 JOHN DOE
ADD PATIENT 321 JOE SCMOE
ADD PATIENT 789 JANE CROW
ADD EXAM 123 444
ADD EXAM 123 445
ADD EXAM 321 446
ADD EXAM 789 447
ADD EXAM 789 601
ADD EXAM 789 602
ADD EXAM 321 447
DEL PATIENT 001
DEL PATIENT 789
DEL PATIENT 789
DEL EXAM 602
ADD EXAM 321 447
ADD EXAM 123 601
DEL EXAM 601
ADD PATIENT 789 JOHN STONE