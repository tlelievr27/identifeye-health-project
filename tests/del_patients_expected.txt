Name: JOHN DOE, Id: 123, Exam Count: 2
Name: JOE SCMOE, Id: 321, Exam Count: 2
Name: JOHN STONE, Id: 789, Exam Count: 0